apps/chat-e2e/src/tests/chatExportImportWithAttachment.test.ts

```diff
@@ -502,3 +509,284 @@ dialTest(
     );
   },
 );
+
+dialTest(
+  'Replay mode: export and import file with attachments in user message',
+  async ({
+    conversationData,
+    fileApiHelper,
+    dataInjector,
+    localStorageManager,
+    dialHomePage,
+    conversations,
+    conversationDropdownMenu,
+    chatBar,
+    confirmationDialog,
+    recentEntitiesAssertion,
+    chat,
+    chatMessages,
+    chatAssertion,
+    chatMessagesAssertion,
+    conversationAssertion,
+    apiAssertion,
+    setTestIds,
+  }) => {
+    setTestIds('EPMRTC-914');
+    let geminiProVisionConversation: Conversation;
+    let gptProVisionConversation: Conversation;
+    let historyConversation: Conversation;
+    let replayConversation: Conversation;
+    let exportedData: UploadDownloadData;
+
+    await dialTest.step(
+      'Prepare conversation with image, pdf attachments in the requests and replay conversation based on it',
+      async () => {
```

Comment:
The code for preparing conversations with attachments is repeated for both models. Probably we can refactor it into a separate function

Reply:
done